configs/rfnext/metafile.yml

```diff
@@ -0,0 +1,232 @@
+Models:
```

Comment:
Please create a `Collection` named `RF-Next`. You can refer to https://github.com/open-mmlab/mmdetection/blob/master/configs/faster_rcnn/metafile.yml#L1-L20